lib/get_process_mem.rb

```diff
@@ -20,6 +20,14 @@ class GetProcessMem
 
   RUNS_ON_WINDOWS = Gem.win_platform?
 
+  def self.preferred_pss?
+    @preferred_pss ||= false
+  end
```

Comment:
めちゃくちゃ小さいんですが、自己代入時のboolean処理は

```suggestion
  def self.preferred_pss?
    return @preferred_pss unless @preferred_pss.nil?
    @preferred_pss = false
  end
```
かと思います！

Reply:
ソウデシタ_(┐「ε:)_